CRM/Utils/Hook.php

```diff
@@ -2133,11 +2133,12 @@ public static function angularModules(&$angularModules) {
    *
    * @code
    * function example_civicrm_alterAngular($angular) {
-   *   $angular->add(ChangeSet::create('mychanges')
+   *   $changeSet = \Civi\Angular\ChangeSetChangeSet::create('mychanges')
```

Comment:
@michaelmcandrew you got too many ChangeSet i think it should be `\Civi\Angular\ChangeSet::create`

Reply:
Ch-ch-ch-ch-changes.... sounds like a song...